app/src/main/java/com/lateinit/rightweight/ui/calendar/CalendarViewModel.kt

```diff
@@ -92,42 +53,27 @@ class CalendarViewModel @Inject constructor(
         currentMonth.value = YearMonth.from(date)
     }
 
-    private fun getSelectedDayInfo(date: LocalDate): ParentDayUiModel? {
+    private fun getSelectedDayInfo(date: LocalDate): HistoryUiModel? {
         return if (date in dateToExerciseHistories.value) {
             getSelectedHistory(date)
         } else {
-            getSelectedRoutineDay(date)
+            _routineTitle.value = DEFAULT_ROUTINE_TITLE
+            _exerciseTime.value = DEFAULT_EXERCISE_TIME
+            null
         }
     }
 
     private fun getSelectedHistory(date: LocalDate): HistoryUiModel? {
         return dateToExerciseHistories.value[date]?.let {
-            _routineTitle.value = it.history.routineTitle
-            _exerciseTime.value = it.history.time
-            it.toHistoryUiModel()
-        }
-    }
-
-    private fun getSelectedRoutineDay(date: LocalDate): DayUiModel? {
-        val today = LocalDate.now()
-        _exerciseTime.value = DEFAULT_EXERCISE_TIME
-
-        return if (date.isBefore(today) || todayRoutineDayPosition == DAY_POSITION_NONE) {
-            _routineTitle.value = DEFAULT_ROUTINE_TITLE
-            null
-        } else {
-            val dayDiff = Period.between(today, date).days + todayRoutineDayPosition
-            val currentRoutineDayPosition =
-                dayDiff % currentRoutineDays.value.size
-
-            _routineTitle.value = currentRoutine?.title ?: DEFAULT_ROUTINE_TITLE
-            currentRoutineDays.value[currentRoutineDayPosition]
+            _routineTitle.value = it.routineTitle
+            _exerciseTime.value = it.time
+            it
```

Comment:
```suggestion
private fun getSelectedHistory(date: LocalDate): HistoryUiModel? {
        return dateToExerciseHistories.value[date].let {
            _routineTitle.value = it?.routineTitle ?: DEFAULT_ROUTINE_TITLE
            _exerciseTime.value = it?.time ?: DEFAULT_EXERCISE_TIME
            it
        }
    }
```
이렇게 두면 getSelectedDayInfo에서 if else로 분기를 나누지 않아도 되지 않을까요?

Reply:
이전엔 분리를 했어야 했는데 변경 되면서 그럴 필요가 없어졌네요?
반영 해보겠습니다!